README.md

```diff
@@ -1,80 +1,81 @@
 ```
 
+Or even:
+
+```bash
+pip install https://github.com/eaedk/testing-zindi-package/archive/master.tar.gz
+```
+
 ## Usage
 
-You can check the [colab notebook here](https://colab.research.google.com/drive/1zzAUWkJ8R5GQzxsdJ5i7XTxaGe2tmUF4?usp=sharing)
+Check out the [demo colab notebook][colab notebook]
 
 ```python
 # create a user object
 from zindi.user import Zindian
 
-my_username = "I_am_Zeus_AI"
-user = Zindian(username = my_username)
-
-#desired output
-[ 🟢 ] 👋🏾👋🏾 Welcome I_am_Zeus_AI 👋🏾👋🏾
+user = Zindian(username="your-username")
 
+# desired output:
+# [ 🟢 ] 👋🏾👋🏾 Welcome your-username 👋🏾👋🏾
 
-user.select_a_challenge()                               # Select a Zindi challenge
+user.select_a_challenge()  # Select a Zindi challenge
 
-user.which_challenge                                    # Get information about the selected challenge
+user.which_challenge  # Get information about the selected challenge
 
-user.leaderboard()                              # Show the Leaderboard of the selected challenge
+user.leaderboard()  # Show the Leaderboard for the selected challenge
 
-user.my_rank                                    # Get the user's leaderboard rank
+user.my_rank  # Get the user's leaderboard rank
 
-user.remaining_subimissions                         # Get information about how many submission you can still push now to Zindi
+user.remaining_submissions  # See how many submission you can still push to Zindi
 
-user.submission_board()                         # Show the user's Submission-board of the selected challenge
+user.submission_board()  # Show the user's Submission-board of the selected challenge
 
-user.download_dataset(destination="tests/dataset") # Download the dataset of the selected challenge
+user.download_dataset(destination="tests/dataset")  # Download the dataset for the selected challenge
 
-user.submit(filepaths=['tests/dataset/SampleSubmission.csv'], comments=['initial submission']) # Push a submission to Zindi : the SampleSubmission file
+# Push a submission to Zindi
+user.submit(filepaths=['tests/dataset/SampleSubmission.csv'],
+            comments=['initial submission']) 
 
-user.remaining_subimissions                             # Get information about how many submission you can still push now to Zindi
+user.remaining_subimissions  # Check how many submissions you still have
 
-user.submission_board()                             # Show the Submission-board of the selected challenge
-
-user.create_team(team_name="New Team")             # Create a team for the selected challenge
+user.submission_board()  # View your updated Submission-board
 
+user.create_team(team_name="New Team")  # Create a team
 ```
 
 ## Status
 
-In progress 🚧 . All listed functions in **Usage** part are working, but please share issues with use and request features that you need.
-
-## Documentation
-
-We will update this table soon ... Now you can refer to the notebook and **Usage** section.
-
```

Comment:
The documentation can be hosted elsewhere e.g. at <https://readthedocs.org/>. This section can be replaced with a link to the docs.